gen3/neural-networks/advanced-examples/ocr/ocr/main.py

```diff
@@ -1,77 +1,113 @@
 import depthai as dai
-import blobconverter
+from depthai_nodes import ParsingNeuralNetwork
+from utils.host_process_detections import ProcessDetections
+from utils.host_sync import DetectionsRecognitionsSync
+from utils.annotation_node import OCRAnnotationNode
+from utils.arguments import initialize_argparser
+from pathlib import Path
 
-from host_east import East
-from host_process_detections import ProcessDetections
-from host_ocr import OCR
-from detections_recognitions_sync import DetectionsRecognitionsSync
+_, args = initialize_argparser()
 
-FPS = 10
+visualizer = dai.RemoteConnection()
+device = dai.Device( dai.DeviceInfo(args.device) if args.device else dai.DeviceInfo())
+platform = device.getPlatform()
 
-detection_model_description = dai.NNModelDescription(modelSlug="east-text-detection", platform="RVC2", modelVersionSlug="256x256")
-detection_archive_path = dai.getModelFromZoo(detection_model_description)
-detection_nn_archive = dai.NNArchive(detection_archive_path)
-
-recognition_model_description = dai.NNModelDescription(modelSlug="text-recognition", platform="RVC2", modelVersionSlug="0012")
-recognition_archive_path = dai.getModelFromZoo(recognition_model_description)
-recognition_nn_archive = dai.NNArchive(recognition_archive_path)
-
-with dai.Pipeline() as pipeline:
+FPS = 5
```

Comment:
Is this temporary while we wait for some DAI/Visualizer improvements?

Reply:
No, this is just because RVC2 is slower and there is no way to have Camera FPS be dynamic. The current practice is to first run the pipeline a couple of times with different FPS to see what works. Setting to 5 FPS leaves 200 miliseconds for RVC2 to detections, `n` recognitions and other processes.